runbotics/runbotics-desktop/src/core/bpm/desktop-runner/desktop-runner.service.ts

```diff
@@ -119,7 +119,7 @@ export class DesktopRunnerService implements OnModuleInit {
         let currentExtensionName: string;
         try {
             const extensions = readdirSync(this.serverConfigService.extensionsDirPath, { withFileTypes: true })
-                .filter((dirent) => dirent.isDirectory())
+                .filter((dirent) => dirent.isDirectory() && dirent.name !== 'common' && dirent.name[0] !== '.')
```

Comment:
typo `dirent`

Reply:
maybe use `startsWith` instead of 0 index

Reply:
what typo? 🤔 
![image](https://github.com/runbotics/runbotics/assets/61866178/2dd496f4-5522-4de9-8373-9e0a17e5ecd9)
![image](https://github.com/runbotics/runbotics/assets/61866178/0a445b1c-90c2-46a0-9289-94309729ff0a)